azure-functions/README.md

```diff
@@ -0,0 +1,22 @@
+# Azure Function Examples
+
+## Incoming Messages
```

Comment:
Are there prerequisites for running these functions? (Such as creating an Event Hub that's linked to ADT, setting up IoT Hub data flow, having certain twins in your instance)
 If so, we should probably list them.